apps/frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,25 @@
+import { useLocation } from "react-router-dom";
+
+import { AppRoute, sidebarLinkIcon } from "~/libs/enums/enums.js";
+
+import { SidebarLink } from "./components/sidebar-link/sidebar-link.js";
+import styles from "./styles.module.css";
+
+const Sidebar: React.FC = () => {
+	const { pathname } = useLocation();
+	return (
+		<div className={styles["sidebarContainer"]}>
+			<div className={styles["logoContainer"]}>Logo</div>
+			<div className={styles["buttonsContainer"]}>
+				<SidebarLink
+					icon={sidebarLinkIcon.MY_WHEEL}
+					pathname={pathname}
+					title="My Wheel"
```

Comment:
Create enum for that

Reply:
You mean moving title to already existing enum? Im actually al little confused whether storing icon/button details as an enums satisfies the quality criteria, because as I understand it you suggested remaking this enum into a map in the previous comment

Reply:
I was talking about moving title to enum, but actually let's create an array with sidebar items. Each item should have title, href and icon

Reply:
where should I store this array? In the same file?